source/game/field/obj/ObjectBreakable.hh

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "game/field/obj/ObjectCollidable.hh"
+
+namespace Field {
+
+class ObjectBreakable : public ObjectCollidable {
+public:
+    ObjectBreakable(const System::MapdataGeoObj &params) : ObjectCollidable(params) {}
+    ~ObjectBreakable() = default;
```

Comment:
Fine for these to be in the header now, but can you make sure you add the addresses hre?
```suggestion
    /// @addr{0x8076EBE0}
    ObjectBreakable(const System::MapdataGeoObj &params) : ObjectCollidable(params) {}
    
    /// @addr{0x8076EC28}
    ~ObjectBreakable() = default;
```